etc/disable-programs.inc

```diff
@@ -452,6 +453,7 @@ blacklist ${HOME}/.local/share/ocenaudio
 blacklist ${HOME}/.local/share/okular
 blacklist ${HOME}/.local/share/orage
 blacklist ${HOME}/.local/share/org.kde.gwenview
+blacklist ${HOME}/.local/share/rhythmbox
```

Comment:
rhythmbox.profile will need to be updated

Reply:
Upps Forgotten, coming soon.

Reply:
Sorry, but now is there